webui/react/src/ioTypes.ts

```diff
@@ -170,10 +170,10 @@ const ioMetricSummary = io.type({
   ]),
 });
 
-export const ioSummaryMetrics = io.partial({
-  avg_metrics: io.record(io.string, ioMetricSummary),
-  validation_metrics: io.record(io.string, ioMetricSummary),
-});
+export const ioSummaryMetrics = io.record(
+  io.string,
+  io.union([io.record(io.string, ioMetricSummary), io.null]),
```

Comment:
Not sure why but encountered the fact the summary metrics may return something like this:
```
{ avg_metrics: null }
```

Reply:
It might be more clear to use the `optional` util function

Reply:
observed that they are typically on older experiments, so hence it's very possible that the metrics are not summarized for them, hence the `null` as a result

Reply:
IMO we should define a `nullable` type rather than using `optional` since `undefined` is not applicable here.